### PR TITLE
Update details around blog contribution

### PR DESCRIPTION
**Pull request by sftim**
Revise https://github.com/kubernetes/community/blob/master/sig-docs/blog-subproject/README.md and some associated pages.

Notification to: @mrbobbytables @onlydole @nate-double-u

### CODE REVIEW

**Comment by natalisucks** [comment]
A general LGTM from me, but checking that @mrbobbytables wants to be removed as an approver before we merge

**Comment by sftim** [comment]
Whoops, did I miss @mrbobbytables ? I shall reinstate.

**Comment by sftim** [comment]
Bob's still there - unless I missed something.

File: sig-docs/blog-subproject/README.md [hunk 1]
```diff
@@ -1,94 +1,108 @@
...
-- **Technical Editors:** [Bob Killen](https://github.com/mrbobbytables), [Taylor Dolezal](https://github.com/onlydole), [Tim Bannister](https://github.com/sftim), [Nate Waddington](https://github.com/nate-double-u)
+<!-- GitHub username alphabetical order -->
+- **Blog approvers:** [Bob Killen](https://github.com/mrbobbytables), [Taylor Dolezal](https://github.com/onlydole),
+  [Nate Waddington](https://github.com/nate-double-u), [Tim Bannister](https://github.com/sftim)
+
+- **Blog shadow approvers:** _no contributors_
+
+- **Blog editors:** _no contributors_
+
+- **Blog shadow editors:** _no contributors_
```
Comment (sftim) [inline sig-docs/blog-subproject/README.md:22]: Same list of people, new job title.

**Review by nate-double-u — COMMENTED** [review]
One nit, otherwise looks good, thx Tim.
/lgtm

File: sig-docs/blog-subproject/README.md [hunk 2]
```diff
@@ -1,94 +1,108 @@
...
 - Kubernetes Partner OSS integration
-- **Original content only**
 
-Unsuitable Content:
+Unsuitable content:
 
 - Vendor product pitches
 - Partner updates without an integration and customer story
-- Syndicated posts (language translations ok)
+- Syndicated posts (it's OK to localized existing articles from English)
```
Comment (nate-double-u) [inline sig-docs/blog-subproject/README.md:56]: nit:
```suggestion
Syndicated posts (it's OK to localize existing articles from English)
```

**Comment by natalisucks** [comment]
/lgtm
@mrbobbytables would you mind giving your approve here? we're working on ramping up our blog contributors to help Tim and Nate's burden asap – thanks!

**Review by mrbobbytables — COMMENTED** [review]
thanks!
/approve

**Pull request merged**